docs/pages/careers.tsx

```diff
@@ -170,6 +212,26 @@ const openRolesData = [
       },
     ],
   },
+  {
+    title: 'Developer Experience',
+    roles: [
+      {
+        title: 'Developer Advocate',
+        description: 'You will own the marketing efforts at MUI.',
```

Comment:
```suggestion
        description: 'You will educate users on the latest features, craft high-quality examples, and demos, engage with the community, write documentation, advocate for creating faster and more appealing UIs, and help to promote the advanced components.',
```

From the description in the role itself.

Reply:
Right, good point, a copy and paste mistake.